src/Core/Localization/Specification/NumberInterface.php

```diff
@@ -34,6 +34,13 @@
  */
 interface NumberInterface
 {
+    /**
+     * Latin numbering system is the "occidental" numbering system. Number digits are 0123456789.
+     * This is the default numbering system in PrestaShop, even for arabian or asian languages, until we
+     * provide a way to configure this in admin.
+     */
+    const NUMBERING_SYSTEM_LATIN = 'latn';
```

Comment:
Already in src/Core/Localization/Locale.php :thinking:

Reply:
I know but I wanted to avoid coupling the two classes..